api/README.md

```diff
@@ -14,7 +14,7 @@ To install:
 To run: `node app.js` (or, say, [`nodemon app.js`](https://github.com/remy/nodemon))
 
 ### API Documentation and Examples
-Documentation for this API is at [https://pages.18f.gov/fbopen/](https://pages.18f.gov/fbopen/), but here are a couple quick sample queries to get you started:
```

Comment:
I'm pretty sure this API isn't even running anymore... if we're going to change the docs to anything it should say something about how the API is no longer up

Reply:
Will make that more obvious.